apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/grpc/Retriable.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.stargate.sgv2.api.common.grpc;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import io.stargate.bridge.proto.StargateBridge;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import javax.inject.Qualifier;
+
+/** Qualifier for retriable {@link StargateBridge}. */
+@Qualifier
+@Retention(RUNTIME)
+@Target({METHOD, FIELD, PARAMETER, TYPE})
+public @interface Retriable {}
```

Comment:
Would it make sense to use name like `RetriableQualifier` as name `Retriable` would usually imply interface for capability, instead of annotation?

Reply:
I kept the java naming standards here, there is no `Qualifier` suffix in the `javax.enterprise.inject.Default` or `javax.enterprise.inject.Any`

Reply:
but I can move it to the dedicated package

Reply:
No problem, I think package is fine.